packages/client/hmi-client/src/workflow/ops/optimize-ciemss/tera-optimize-ciemss-node.vue

```diff
@@ -137,13 +137,15 @@ const startForecast = async (simulationIntervetions) => {
 			start: 0,
 			end: props.node.state.endTime
 		},
-		interventions: simulationIntervetions,
 		extra: {
 			num_samples: props.node.state.numSamples,
 			method: props.node.state.solverMethod
 		},
 		engine: 'ciemss'
 	};
+	if (simulationIntervetions.length > 0) {
```

Comment:
typo interveNtions. and use isEmpty

Reply:
no one said var names need to be english :O